pipes.Rmd

```diff
@@ -37,26 +35,51 @@ If the arguments to a function don't all fit on one line, put each argument on
 its own line and indent:
 
 ```{r, eval = FALSE}
-iris %>%
-  group_by(Species) %>%
+# Good
+iris |>
   summarise(
     Sepal.Length = mean(Sepal.Length),
-    Sepal.Width = mean(Sepal.Width)
+    Sepal.Width = mean(Sepal.Width),
+    .by = Species
   )
+
+# Bad
+iris |>
+  summarise(Sepal.Length = mean(Sepal.Length), Sepal.Width = mean(Sepal.Width), .by = Species)
+```
+
+For data analysis, we recommend using the pipe whenever a function needs to span multiple lines, even if it's only a single step.
+
+```{r, eval = FALSE}
+# Bad
+summarise(
+  iris,
+  Sepal.Length = mean(Sepal.Length),
+  Sepal.Width = mean(Sepal.Width),
+  .by = Species
+)
 ```
 
 ## Short pipes
 
-A one-step pipe can stay on one line, but unless you plan to expand it later on, you should consider rewriting it to a regular function call.
+It's ok to write a short pipe on a single line:
```

Comment:
nit: similar to above, this feels a bit more substantive than switching `%>%` -> `|>`